src/istio/README.md

```diff
@@ -1 +1,11 @@
 # Istio
+
+A powerful service mesh tool that provides traffic management, load balancing, security, and observability features.
+
+## Gateways
+
+UDS Core provides a few Istio [Gateway](https://istio.io/latest/docs/reference/config/networking/gateway/) resources to allow ingress into the service mesh. Each one serves a different purpose and can be used to route traffic to different services.
+
+1. **(Required)** Tenant Gateway - This gateway provides ingress to typical end-user applications. By default, UDS Core deploys a few services on this gateway, such as the Keycloak SSO portal. This gateway is typically exposed to end users of the applications deployed on top of UDS Core.
+2. **(Required)** Admin Gateway - This gateway provides ingress to admin-related applications that are not for use by the default end user. By default, UDS Core deploys a few services on this gateway, such as the Admin Keycloak interface. This gateway is typically accessible to admins of the applications deployed on top of UDS Core. *Since the Admin and Tenant Gateways are logically separated, it is possible to have different security controls on each gateway.*
+3. **(Optional)** Passthrough Gateway - This gateway allows mesh ingress without TLS termination performed by Istio. This could be useful for applications that need to (or currently) handle their own TLS termination. This gateway used to be a default component of UDS Core but is no longer deployed by default. To deploy this gateway, you must specify `istio-passthrough-gateway` as an `optionalComponent` in your UDS Bundle configuration.
```

Comment:
This readme doesn't get picked up on our docs site (https://uds.defenseunicorns.com/docs/) since that only grabs from the `docs/` subfolder. I wonder if it would make sense to add a new file under the [configuration folder](https://uds.defenseunicorns.com/core/configuration/) for Istio/"ingress"? We could start that file in this PR by listing this gateway information + example for the passthrough gateway optionalComponent, and I think it would also make sense to list out some of the config + examples for domain and cert values either in this PR or in follow on updates.

Reply:
I can definitely port this information over into the docs site.  I will also add an example for adding istio passthrough gateway as an optional component.  

@mjnagel As for overriding the cert and domain values... at what layer is the end user recommended to set these?  Is this possible via a uds-config file or do you literally override the helm values set in a file like [this](https://github.com/defenseunicorns/uds-core/blob/main/src/istio/values/config-admin.yaml) one?

Reply:
Yeah for domain/cert the path we typically recommend is with bundle overrides + uds-config file. [This](https://github.com/defenseunicorns/uds-core/blob/main/bundles/k3d-standard/uds-bundle.yaml#L76-L93) is a good example of the override section needed to expose the TLS cert/key pairs as config, then someone could set `ADMIN_TLS_CERT` and the others via config file or the [other methods for variables](https://uds.defenseunicorns.com/cli/overrides/#variables).

For domain, it is exposed as a zarf variable ([here](https://github.com/defenseunicorns/uds-core/blob/main/src/istio/chart/values.yaml#L5)) and typically is set via the `shared` key in your uds config (see example [here](https://uds.defenseunicorns.com/cli/quickstart-and-usage/#variables-and-configuration)).

Reply:
Also just to point you in the right direction if it wasn't clear, [this is the config folder](https://github.com/defenseunicorns/uds-core/tree/main/docs/configuration) from the docs site and all docs should be markdown with a hugo flavor (ex: [frontmatter](https://gohugo.io/content-management/front-matter/) for title, etc).

Reply:
Updated.  Will need a thorough read through to make sure I captured everything correctly.

Reply:
LGTM overall, added 2 formatting comments and one on the content (envs + config file). Resolving this thread in favor of the new ones.